export const failwith = (message: string) => {
  throw new Error(message);
};
